Library/Formula/swiftcov.rb

```diff
@@ -0,0 +1,33 @@
+class Swiftcov < Formula
+  homepage "https://github.com/realm/SwiftCov"
+  desc "A tool to generate test code coverage information for Swift."
+  url "https://github.com/realm/SwiftCov.git", :tag => "0.1.0", :revision => "3a13f0ede4449fd3e4e07878031544d906ee19f9"
```

Comment:
`:tag` and `:revision` should be pushed onto unique lines below so the formula isn't overly wide.